### DISCUSSION TITLE
RoundedTB Support integration

### DISCUSSION THREAD

**Discussion by ShiroUwUs7** [valinet/ExplorerPatcher#62] [Ideas]
Support for RoundedTB suggestion, to still give the feel of windows 11 with the support of windows 10 added in.

**Comment by valinet** [comment]
Does RoundedTB work in Windows 10 as well?

**Reply by ShiroUwUs7** [reply]
It does. closest i've gotten is this: 
![unknown 1](https://user-images.githubusercontent.com/53096166/136713706-1b38ffd7-2690-40ea-aa95-37f438c37d5f.png)

vs windows 10 on the right, win 11 on the left.

**Reply by valinet** [reply]
Okay, understand. So RoundedTB is kindof broken on Windows 11 with the classic taskbar, right? Can you post to their GitHub as well, maybe they have a look at it on their side...? Eventually, I hope I will look into implementing something along these lines here as well. Thanks.

**Reply by ShiroUwUs7** [reply]
https://github.com/torchgm/RoundedTB/  Correct I have to set it a certian way, and if the tray icon is hidden then it breaks, and the other side of the taskbar is perm hidden till I end the program and restart exploerer.exe as even enabling it again doesn't bring it back. 
![LEKAfdL 1](https://user-images.githubusercontent.com/53096166/136713887-e66320db-912e-4407-bada-80150000152e.png)

I have to enable this (desktop taskbar) to keep it stable. 
![image](https://user-images.githubusercontent.com/53096166/136713841-5b1db002-0b22-441a-9913-90194c8fbe53.png)

**Reply by valinet** [reply]
Yeah, open an issue there. The Windows 11 classic taskbar is not actually exactly the one from the latest Windows 10 stable build, it is more like the latest code available in Insider Preview before they decided to kill it off and create Windows 11. They were preparing some changes here and there (like the control center icon), so if Windows 11 would have never released and only an update to Windows 10 with the same shell, probably RoundedTB would have had the same problem. I think it is a problem with that program, in that something slightly changed in Windows and they just need to update it to work with it, hopefully. Please talk to the people there as well, maybe they can help more, and even fix the software.
